File: Rock, Paper, Scissors/RPS.js
const playerMove = pMove => {
	return pMove;
};

const compMove = () => {
	let number = Math.floor(Math.random() * 3) + 1;
	let cMove = '';
	console.log(number);
	if (number === 1) {
		cMove = 'Rock';
	} else if (number === 2) {
		cMove = 'Scissors';
	} else if (number === 3) {
		cMove = 'Paper';
	} else {
		console.log('That isn not a correct move');
	}
  console.log(cMove);
	return cMove;
};

const playGame = (pMove, cMove) => {
	if (pMove === 'Rock' && cMove === 'Scissors') {
		console.log('Player wins!');
	} else if (pMove === 'Rock' && cMove === 'Paper') {
		console.log('Computer wins!');
	} else if (pMove === 'Paper' && cMove === 'Scissors') {
		console.log('Computer wins!');
	} else if (pMove === 'Paper' && cMove === 'Rock') {
		console.log('Player wins!');
	} else if (pMove === 'Scissors' && cMove === 'Rock') {
		console.log('Computer wins!');
	} else if (pMove === 'Scissors' && cMove === 'Paper') {
		console.log('Player wins!');
	} else {
		console.log('It is a tie!');

	}
};

playGame(playerMove('Rock'), compMove());

File: Rock, Paper, Scissors/RPS.rb
def player_guess
	puts "Enter Rock, Paper, or Scissors"
	@choice = gets.chomp
	@choice.capitalize!
end

def comp_guess
	num = 1 + rand(3)
	if num == 1
		@guess = "Rock"
	elsif num == 2
		@guess = "Paper"
	elsif num == 3
		@guess = "Scissors"
	end
end

def game_play(comp, player)
	bool = false
	loop do
		if @guess == "Rock" && @choice == "Scissors"
			puts "Computer wins!"
			bool = true
		elsif @guess == "Rock" && @choice == "Paper"
			puts "Player wins!"
		elsif @guess == "Paper" && @choice == "Rock"
			puts "Computer wins!"
			bool = true
		elsif @guess == "Paper" && @choice == "Scissors"
			puts "Player wins!"
		elsif @guess == "Scissors" && @choice == "Paper"
			puts "Computer wins!"
			bool = true
		elsif @guess == "Scissors" && @choice == "Rock"
			puts "Player wins!"
		elsif @guess == @choice
			puts "It's a tie!"
		end
		break if bool == true
		player_guess
		puts "Computer chose #{@guess}"
	end
end
game_play(comp_guess, player_guess)
